Cargo.toml

```diff
@@ -168,7 +168,8 @@ rustls-pemfile = "2.2"
 serde = "1.0"
 serde_ignored = "0.1"
 serde_json = "1.0"
-sha-1 = "0.10"
+serial_test = "3.2"
```

Comment:
It was removed by #3482 and revoked here accidentally?

Reply:
Good catch, fixed now